setup.py

```diff
@@ -35,7 +35,7 @@
     author="Adafruit Industries",
     author_email="circuitpython@adafruit.com",
     install_requires=[
-        "pyserial",
+        "pyserial~=3.5",
```

Comment:
Shouldn't it be this for minimum version?

```suggestion
        "pyserial>=3.5",
```

Reply:
`~` is for "compatible release": https://www.python.org/dev/peps/pep-0440/#compatible-release
So `pyserial~=3.5` would be the same as `"pyserial>=3.5,<4"`

Reply:
:+1: Didn't know that!